libs/eo/landing-page/shell/src/lib/eo-landing-page-shell.component.ts

```diff
@@ -37,13 +37,16 @@ const selector = 'eo-landing-page-shell';
       }
     `,
   ],
-  template: `<eo-landing-page-header></eo-landing-page-header>`,
+  template: `
+    <eo-landing-page-header></eo-landing-page-header>
+    <eo-landing-page-footer></eo-landing-page-footer>
+  `,
```

Comment:
praise: Beautiful! 👏 No presentational logic in the shell except the order of child components.